presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestPrestoContainerBasicQueries.java

```diff
@@ -0,0 +1,132 @@
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+public class TestPrestoContainerBasicQueries
+{
+    private static final String PRESTO_COORDINATOR_IMAGE = System.getProperty("coordinatorImage", "presto-coordinator:latest");
+    private static final String PRESTO_WORKER_IMAGE = System.getProperty("workerImage", "presto-worker:latest");
+    private static final String BASE_DIR = System.getProperty("user.dir");
+    private static final String CONTAINER_TIMEOUT = System.getProperty("containerTimeout", "120");
+    private static final Network network = Network.newNetwork();
+    private GenericContainer<?> coordinator;
+    private GenericContainer<?> worker;
+
+    @BeforeClass
+    public void setUp()
+            throws InterruptedException
+    {
+        coordinator = new GenericContainer<>(PRESTO_COORDINATOR_IMAGE)
+                .withExposedPorts(8081)
+                .withNetwork(network).withNetworkAliases("presto-coordinator")
+                .withFileSystemBind(BASE_DIR + "/testcontainers/coordinator/etc", "/opt/presto-server/etc", BindMode.READ_WRITE)
+                .withFileSystemBind(BASE_DIR + "/testcontainers/coordinator/entrypoint.sh", "/opt/entrypoint.sh", BindMode.READ_ONLY)
+                .waitingFor(Wait.forLogMessage(".*======== SERVER STARTED ========.*", 1))
+                .withStartupTimeout(Duration.ofSeconds(Long.parseLong(CONTAINER_TIMEOUT)));
+
+        worker = new GenericContainer<>(PRESTO_WORKER_IMAGE)
+                .withExposedPorts(7777)
+                .withNetwork(network).withNetworkAliases("presto-worker")
+                .withFileSystemBind(BASE_DIR + "/testcontainers/nativeworker/velox-etc", "/opt/presto-server/etc", BindMode.READ_ONLY)
+                .withFileSystemBind(BASE_DIR + "/testcontainers/nativeworker/entrypoint.sh", "/opt/entrypoint.sh", BindMode.READ_ONLY)
+                .waitingFor(Wait.forLogMessage(".*Announcement succeeded: HTTP 202.*", 1));
+
+        coordinator.start();
+        worker.start();
+
+        // Wait for worker to announce itself.
+        TimeUnit.SECONDS.sleep(5);
+    }
+
+    @AfterClass
+    public void tearDown()
+    {
+        coordinator.stop();
+        worker.stop();
+    }
+
+    private Container.ExecResult executeQuery(String sql)
+            throws IOException, InterruptedException
+    {
+        // Command to run inside the coordinator container using the presto-cli.
+        String[] command = {
+                "/opt/presto-cli",
+                "--server",
+                "presto-coordinator:8081",
+                "--catalog",
+                "tpch",
```

Comment:
Might be great to allow both catalog and schema to be sent as parameters to this function.